docs/buildah-inspect.md

```diff
@@ -0,0 +1,27 @@
+## buildah-mount "1" "May 2017" "buildah"
+
+## NAME
+buildah inspect - Display information about a working container.
+
+## SYNOPSIS
+**buildah** **inspect** [*options* [...] --] **containerID**
+
+## DESCRIPTION
+Prints information about a working container's configuration.
+
+## OPTIONS
+
+**--format** *template*
+
+Use *template* as a Go template when formatting the output.
```

Comment:
Are there any example templates we could include here and/or other documentation?